landscape/client/tests/test_deployment.py

```diff
@@ -601,3 +602,33 @@ def test_generate_computer_title_with_date(
             },
         )
         self.assertEqual(title, "2024-machine")
+
+
+class ArgConversionTest(LandscapeTest):
+    """Tests for `convert_arg_to_bool` function"""
+
+    def test_true_values(self):
+        TRUTHY_VALUES = {"true", "yes", "y", "1", "on", "TRUE", "Yes"}
+        val = True
+        for t in TRUTHY_VALUES:
+            val = convert_arg_to_bool(t)
+        self.assertTrue(val)
+
+    def test_false_values(self):
+        FALSY_VALUES = {"false", "no", "n", "0", "off", "FALSE", "No"}
+        val = False
+        for f in FALSY_VALUES:
+            val = convert_arg_to_bool(f)
+        self.assertFalse(val)
```

Comment:
We'll want this in the loop - it's OK if the test fails after the first failing instance. We don't have to worry about subtests.

You also won't need to initialize `val`, then.